packages/core/ui-settings/core-ui-settings-common/src/timezones.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import moment from 'moment-timezone';
+
+export const TIMEZONE_OPTIONS = [
+  ...moment.tz
+    .names()
+    // We need to filter out some time zones, that moment.js knows about, but Elasticsearch
+    // does not understand and would fail thus with a 400 bad request when using them.
+    .filter((tz) => !['America/Nuuk', 'EST', 'HST', 'ROC', 'MST'].includes(tz)),
```

Comment:
Do you want to exclude ['America/Nuuk', 'EST', 'HST', 'ROC', 'MST'] here or to keep them? Now you are keeping them. Or I'm missing something?